lua/nvim-lsp-ts-utils/inlay-hints.lua

```diff
@@ -0,0 +1,91 @@
+local o = require("nvim-lsp-ts-utils.options")
+
+local M = {}
+
+M.state = {
+    enabled = false,
+    ns = vim.api.nvim_create_namespace("ts-inlay-hints"),
+}
+
+function M.setup_autocommands()
+    vim.cmd([[
+       augroup TSInlayHints
+       au BufEnter,BufWinEnter,TabEnter,BufWritePost,TextChanged,TextChangedI *.ts,*.js,*.tsx,*.jsx :lua require'nvim-lsp-ts-utils'.autocmd_fun()
+       augroup END
+   ]])
+end
+
+local function handler(err, result, ctx)
+    if not err and result and M.state.enabled then
+        vim.api.nvim_buf_clear_namespace(ctx.bufnr, 0, 0, -1)
```

Comment:
```suggestion
        vim.api.nvim_buf_clear_namespace(ctx.bufnr, 0, M.state.ns, -1)
```

I'm not too familiar with the API, but this should also specify the namespace, right? At the moment it's messing with indent-blankline.

Reply:
yes, we should use the hide function here